data/src/main/java/nl/tehdreamteam/se42/data/EntityManagerLocator.java

```diff
@@ -0,0 +1,20 @@
+package nl.tehdreamteam.se42.data;
+
+import javax.persistence.EntityManager;
+import javax.persistence.Persistence;
+
+/**
+ * @author Oscar de Leeuw
+ */
+public class EntityManagerLocator {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.tehmessiah.nl/coding_rules#rule_key=squid%3AS1118)